components/dashboard/package.json

```diff
@@ -97,7 +97,7 @@
     "react-scripts": "^5.0.1",
     "tailwind-underline-utils": "^1.1.2",
     "tailwindcss-filters": "^3.0.0",
-    "typescript": "^5.4.5",
+    "typescript": "^5.5.4",
```

Comment:
doing this because I was using a feature introduced in 5.5

Reply:
Just curious, which feature?

Reply:
It's the [Inferred Type Predicates](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-5.html#inferred-type-predicates), which lets us do `filter` with `!== undefined` without having to use something like `repo is SuggestedRepository`, basically typescript being smart about the fact that when you check something isn't undefined, its type correctly changes.